fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepaymentScheduleInstallment.java

```diff
@@ -488,6 +490,10 @@ public void resetChargesCharged() {
         this.penaltyCharges = null;
     }
 
+    public boolean isCurrentInstallment(LocalDate transactionDate) {
+        return getFromDate().isBefore(transactionDate) && !getDueDate().isBefore(transactionDate);
```

Comment:
If this is the 1st installment, it should be inclusive.

Reply:
removed the method instead, as it's no longer needed..